incubator/hnc/config/crd/patches/config_singleton_enum.yaml

```diff
@@ -0,0 +1,5 @@
+            properties:
```

Comment:
If these are only used by the `hack` scripts, I'd prefer to put this in `hack` as well, or maybe even as a heredoc in the script itself, rather than having a bunch of extra files floating around that don't make a lot of sense in isolation.

Reply:
Done, added a comment and moved everything into `hack/crd_patches`.